### ISSUE TITLE
User request - Add location description

### ISSUE THREAD

**Issue by nathankota** [agrc/roadkill-mobile#325]
From the same user as #324.

> Another way to encourage off-highway reporting might be to include an option for the app user to enter a description of the location. Perhaps including some prompts to enter estimated mile-markers, or other road and landscape features.

**Comment by steveoh** [comment]
Could/are we reverse route and milepost geocoding from the device location to get an accurate marker?

**Comment by stdavis** [comment]
> Could/are we reverse route and milepost geocoding from the device location to get an accurate marker?
No, we are not. That sounds like the opposite of what the user is requesting to me.

**Comment by steveoh** [comment]
> That sounds like the opposite of what the user is requesting to me.

I must have misread the request. It sounds like they want to enter an estimated mile marker... 

It seems that a reverse geocode could provide them a decent place to start from? Like an editable field that could display we think you are here?

**Comment by stdavis** [comment]
I think the problem is that they know the approximate mile marker, but they don't want to stop and create the report on the side of the highway using the GPS. So they know the mile marker, but not the location since they are not at the actual site of the animal when they enter the report.
